sunpy/net/attr.py

```diff
@@ -31,7 +31,7 @@
 
 _ATTR_TUPLE = namedtuple("attr", "name client name_long desc")
 # Matches any number.
-NUMBER_REGEX = re.compile(r"(\d+(?:\.\d+)?)")
+NUMBER_REGEX = re.compile(r"^(\d+$|\d(?:\.\d+)?)")
```

Comment:
What does this match now?

Reply:
The whole phrase if it is compose of only numbers or the first character if it's a number. [Check it out](https://regex101.com/r/8jR8Cb/1).